<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>530</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>DDOK DDOK</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(252, 252, 244)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>530</height>
    </size>
   </property>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="accessibleName">
     <string>startButton</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 230, 153)</string>
    </property>
    <property name="text">
     <string>응 시 하 기</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>250</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="warning_msg">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
     </font>
    </property>
    <property name="text">
     <string>시험에 응시할 수 없습니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="recog_title">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어 Bold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>본 인 확 인</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="notice">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어 Light</family>
     </font>
    </property>
    <property name="text">
     <string>정면의 카메라를 응시해주세요.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="match_true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>250</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(1, 218, 1)</string>
    </property>
    <property name="text">
     <string>일        치</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="match_false">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>250</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(191, 6, 9)</string>
    </property>
    <property name="text">
     <string>불   일   치</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>4</number>
    </property>
   </widget>
   <zorder>warning_msg</zorder>
   <zorder>startButton</zorder>
   <zorder>img_frame</zorder>
   <zorder>recog_title</zorder>
   <zorder>notice</zorder>
   <zorder>match_true</zorder>
   <zorder>match_false</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
